ScreenControl_Unity/Assets/ScreenControl/Service/Scripts/Interactions/InteractionModules/InteractionModule.cs

```diff
@@ -59,9 +59,9 @@ void Update()
         // This is the main update loop of the interaction module
         protected virtual void UpdateData(Leap.Hand hand) { }
 
-        protected void SendInputAction(ScreenControlTypes.InputType _inputType, Positions _positions, float _distanceFromScreen, float _progressToClick)
+        protected void SendInputAction(ScreenControlTypes.InputType _inputType, Positions _positions, float _progressToClick)
```

Comment:
How come we're stripping the distance out of this? Do we not need to keep it for the plugin to filter the cursor to limited distances?

Reply:
Distance actually exists in Positions, so we were sending it twice and ignoring one. When the WS version is created, it sends the distance too :)